utils/lpc55-builder/Makefile

```
@@ -68,22 +39,24 @@ provision-release:
 	$(MAKE) bl-erase-firmware
 	lpc55 reboot
 	./scripts/usbwait.sh 1fc9:0021
-	# Step 1: build & flash provisioner, provision keystore
+	# Step 1: build & flash provisioner, provision keystore, cmpa
 	$(MAKE) build FEATURES=provisioner,$(FEATURES)
 	$(MAKE) bl-provision-keystore
+	lpc55 reboot
+	./scripts/usbwait.sh 1fc9:0021
+	$(MAKE) bl-config-cmpa-develop
+	lpc55 reboot
+	./scripts/usbwait.sh 20a0:42dd
 	$(MAKE) bl-flash
```

```suggestion
	$(MAKE) bl-provision-keystore
	lpc55 reboot
	./scripts/usbwait.sh 1fc9:0021
	$(MAKE) bl-config-cmpa-develop
	lpc55 reboot
	./scripts/usbwait.sh 20a0:42dd
	$(MAKE) build FEATURES=provisioner,$(FEATURES)
	$(MAKE) bl-flash
```

Keeping the build directly before the flash makes it easier to see what is flashed.